### PR TITLE
NXBT-2423: parse coma and type check

### PR DESCRIPTION
**Pull request by alexistimic**
https://jira.nuxeo.com/browse/NXBT-2423

### CODE REVIEW

**Comment by artpick** [comment]
👍

File: ondemand-testandpush-metrics.py [hunk 1]
```diff
@@ -79,6 +79,12 @@ def parse_test(line):
         return 0
     seconds = line[idx + len(marker1):line.find(marker2, idx)]
     if seconds:
+        if ',' in seconds:
+            seconds = seconds.replace(',','')
+        try:
+            float(seconds)
```
Comment (LKleinNux) [inline ondemand-testandpush-metrics.py]: Here you can directly do `return float(seconds)`, and remove the one on line 88

File: ondemand-testandpush-metrics.py [hunk 2]
```diff
@@ -79,7 +79,12 @@ def parse_test(line):
         return 0
     seconds = line[idx + len(marker1):line.find(marker2, idx)]
     if seconds:
-        return float(seconds)
+        if ',' in seconds:
+            seconds = seconds.replace(',','')
+        try:
+            return(float(seconds))
```
Comment (BoboTiG) [inline ondemand-testandpush-metrics.py]: `return float(seconds)` is sufficient (no need for parenthesis).

**Comment by artpick** [comment]
👍

File: ondemand-testandpush-metrics.py [hunk 3]
```diff
@@ -79,7 +79,12 @@ def parse_test(line):
         return 0
     seconds = line[idx + len(marker1):line.find(marker2, idx)]
     if seconds:
-        return float(seconds)
+        if ',' in seconds:
+            seconds = seconds.replace(',', '.')
```
Comment (efge) [inline ondemand-testandpush-metrics.py]: I don't think this is correct. `933,915.666` means `933915.666` so you should just remove the `,`, not replace it with a `.`.

Comment (jcarsique) [reply ondemand-testandpush-metrics.py]: fixed in https://github.com/nuxeo/integration-scripts/pull/30
review in progress

**Pull request merged**